framework/src/executioners/EigenExecutionerBase.C

```diff
@@ -319,7 +319,7 @@ EigenExecutionerBase::inversePowerIteration(unsigned int min_iter,
 
     if (echo)
       _console << " ________________________________________________________________________________ "
-               << std::endl;
+               << std::endl << std::flush;
```

Comment:
I don't think this is necessary. I'll have to take a look at what you are seeing. By the very definition of these two methods, you are flushing twice!